pyproject.toml

```diff
@@ -30,7 +30,7 @@ dependencies = [
   "matplotlib~=3.7.1",
   "xarray==0.19.0",
   "numpy~=1.24.3",
-  "archytas~=1.1.5",
+  "archytas @ git+https://github.com/jataware/archytas.git@98e4ed87cc0b08a9b506151f1966f4117811f008",
```

Comment:
Make sure you don't commit this.

Reply:
don't worry, i put this on the pre-merge checklist :+1: